pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/IntegrationTestSuite.java

```diff
@@ -176,7 +176,7 @@ public static void setup(boolean isCachingDisabled) {
     private static void initializeDatabase() {
         RouteTestUtil.loadConfig();
         RouteTestUtil.loadSqlConfig();
-        MySqlTestContainerUtil.initializeTestDbs();
+        DBTestContainer.initializeTestDbs();
```

Comment:
I see this class uses the same setup optimization strategy as the one I recently modified from DSS where we had one set of tests clobbering the other.  Do you think there's any benefit in preemptively removing the early return at line 142?